packages/backend/src/services/ProjectService/ProjectService.test.ts

```diff
@@ -129,4 +140,69 @@ describe('ProjectService', () => {
             expect(result).toEqual(expectedExploreSummaryFilteredByName);
         });
     });
+    describe('getJobStatus', () => {
```

Comment:
These tests should live on the `abilityBuilder` since we want to just test the logic of the ability

Reply:
maybe some, but not the one that check if projectUuid is null and userUuid matches...